alert-policies/temporal/ActivityScheduleStartLatency.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Activity Schedule to Start Latency
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal activity schedule to start latency exceeds 5 seconds for 5 minutes.
```

Comment:
Unsure how to understand this. Can you please clarify this?

Reply:
Hi @seemantk,
The Temporal Schedule-To-Start Timeout represents the maximum allowable duration between scheduling an activity task (placing it in a task queue) and a worker starting (picking it up from the task queue). Essentially, it serves as a restriction on the length of time an activity task can remain in the queue.